MIDIDriver/src/jp/kshoji/driver/midi/activity/AbstractMultipleMidiActivity.java

```diff
@@ -5,7 +5,7 @@
 import android.hardware.usb.UsbDevice;
 import android.hardware.usb.UsbManager;
 import android.os.Bundle;
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
```

Comment:
i am pretty sure we could add some compat library to keep the old imports, but juckt?

Reply:
Does [npx jetifier](https://developer.android.com/studio/command-line/jetifier) not take care of this?